app/src/main/res/layout/button_tip_item.xml

```diff
@@ -22,8 +22,8 @@
         android:lineSpacingExtra="2dp"
         android:maxLines="2"
         android:textAppearance="@style/HeaderTextStyle"
-        android:textColor="@color/primary_text_dark_theme"
-        app:drawableTint="@color/primary_text_dark_theme"
+        android:textColor="@color/photonLightGrey05"
```

Comment:
Opted to inline `photonLightGrey05` since the colors in the background are static and does not depend on a dark/light theme.

Reply:
By static, I mean they are inlined or hard coded to one particular color.